src/public/manifest.json

```diff
@@ -78,7 +78,6 @@
         "name": "vApp Create Extension Point",

```

Comment:
Bump the @vcd/sdk version to this one which has the organisation extension point support.